src/sentry/models/grouplink.py

```diff
@@ -0,0 +1,62 @@
+"""
+sentry.models.groupseen
+~~~~~~~~~~~~~~~~~~~~~~~
+
+:copyright: (c) 2010-2014 by the Sentry Team, see AUTHORS for more details.
+:license: BSD, see LICENSE for more details.
+"""
+from __future__ import absolute_import
+
+from django.db import models
+from django.utils import timezone
+from django.utils.translation import ugettext_lazy as _
+from jsonfield import JSONField
+
+from sentry.db.models import Model, sane_repr, BoundedBigIntegerField, BoundedPositiveIntegerField
+
+
+class GroupLink(Model):
+    """
+    Link a group with an extenal resource like a commit, issue, or pull request
+    """
+    __core__ = False
+
+    class Relationship:
+        unknown = 0
+        resolves = 1
+        links = 2
```

Comment:
what about calling this `references` or something? don't really care either way, just seems like a way to not overuse the word `link`